packages/cryptsetup/Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "cryptsetup"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+[package.metadata.build-package]
+releases-url = "https://www.kernel.org/pub/linux/utils/cryptsetup/v2.7"
+
+[lib]
+path = "../packages.rs"
+
+[[package.metadata.build-package.external-files]]
+url = "https://www.kernel.org/pub/linux/utils/cryptsetup/v2.7/cryptsetup-2.7.5.tar.xz"
+sha512 = "13eca93cdb00a143d2ca60b6f66ede5adc4072ca0c4bfebd8454a3541e69d269fcdb4afc97ad799e87a999b2bd46c1f31fa924a3d616d72a3337970b1e718d55"
+force-upstream = true
+
+[build-dependencies]
+glibc = { path = "../glibc" }
+device-mapper = { path = "../device-mapper" }
+json-c = { path = "../json-c" }
+libblkid = { path = "../util-linux" }
+libuuid = { path = "../util-linux" }
```

`buildsys` only knows about the crate or top-most package name, so this should just be:
```suggestion
util-linux = { path = "../util-linux" }
```